sections/04_study.md

```diff
@@ -52,7 +52,104 @@ particularly notable in this area?*
 
 ### Protein secondary and tertiary structure
 
-*Jinbo Xu is writing this*
+Proteins play fundamental roles in all biological processes including the 
+maintenance of cellular integrity, metabolism, transcription/translation, and 
+cell-cell communication. Complete description of protein structures and 
+functions is a fundamental step towards understanding biological life and 
+also highly relevant in the development of therapeutics and drugs. Tons of 
+protein sequences have been generated, but fewer than 100,000 of them 
+have experimentally-solved structures. As a result, computational structure 
+prediction is essential for a majority number of protein sequences. However, 
+predicting protein 3D structures from sequence alone is very challenging, 
+especially when similar templates are not available. In the past decades, 
+various computational methods have been developed to predict protein 
+structure from different aspects, including prediction of secondary structure, 
+torsion angles, solvent accessibility, inter-residue contact map, disorder 
+regions and side-chain packing.
+
+Machine learning is extensively applied to predict protein structures and 
+some success has been achieved. For example, secondary structure can be 
+predicted with about 80% of Q3 accuracy by a 2-layer neural network 
+method PSIPRED [@doi:10.1093/bioinformatics/16.4.404]. Starting from 
+2012, deep learning has been gradually introduced to protein structure 
+prediction. The adopted deep learning models include deep belief network, 
+LSTM(long short-term memory), deep convolutional neural networks (DCNN) 
+and deep convolutional neural fields[@doi:10.1007/978-3-319-46227-1_1 
+@doi:10.1038/srep18962]. Here we focus on deep learning methods for 
+two representative subproblems: secondary structure prediction and 
+contact map prediction. Secondary structure refers to local conformation of 
+a sequence segment while a contact map contains information of global 
+conformation.
+
+Protein secondary structure can be described by 3 states or 8 states with 
+the latter providing finer-grained information than the former. However, 
+many more methods are developed to predict 3-state secondary structure 
+than 8-state secondary structure. A predictor is typically evaluated by Q3 
+and Q8 accuracy, respectively. Qi et al. developed a multi-task deep learning 
+method to simultaneously predict several local structure properties 
+including secondary structures [@doi:10.1371/journal.pone.0032235]. 
+Cheng group predicted secondary structure using deep belief networks 
+[@doi: 10.1109/TCBB.2014.2343960]. Zhou developed an iterative deep 
+learning framework to simultaneously predict secondary structure, 
+backbone torsion angles and solvent accessibility 
+[@doi:10.1038/srep11476]. However, none of these deep learning methods 
+achieved significant improvement over PSIPRED in terms of Q3 accuracy. In 
```

Comment:
Most likely the failure is due to the deep network architecture. Previous methods do not use deep convolutional neural network. Another reason is that the Q3 accuracy is already very high (~80%) and the estimated limit is about 88%, so it is not easy to improve.